### PR TITLE
[eclipse/xtext#1283] update to MWE 2.10 M2

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1283] update to MWE 2.10 M2

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: releng/org.eclipse.xtext.target/org.eclipse.xtext.target-photon.target [hunk 1]
```diff
@@ -94,7 +94,7 @@
 	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
 		<unit id="org.eclipse.emf.mwe2.lib" version="0.0.0"/>
 		<unit id="org.eclipse.emf.mwe2.launcher.feature.group" version="0.0.0"/>
-		<repository location="http://download.eclipse.org/modeling/emft/mwe/updates/releases/2.9.1/"/>
```
Comment (cdietrich) [inline releng/org.eclipse.xtext.target/org.eclipse.xtext.target-photon.target]: @kthoms this needs to be changed too

Comment (kthoms) [reply releng/org.eclipse.xtext.target/org.eclipse.xtext.target-photon.target]: yep

File: releng/org.eclipse.xtext.target/org.eclipse.xtext.target-latest.target [hunk 2]
```diff
@@ -86,11 +87,6 @@
 		<repository location="http://download.eclipse.org/modeling/emf/emf/builds/nightly/latest"/>
 	</location>
 
-	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
-		<unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
```
Comment (cdietrich) [inline releng/org.eclipse.xtext.target/org.eclipse.xtext.target-latest.target]: ? what has changed here?

Comment (kthoms) [reply releng/org.eclipse.xtext.target/org.eclipse.xtext.target-latest.target]: Moved to be consumed from SimRel. We don't rely on a specific version. SimRel contains org.eclipse.draw2d.feature.group=3.10.100.201606061308

**Pull request merged**